lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -8390,7 +8390,7 @@ using namespace Js;
                 }
                 else
                 {
-                    separator = scriptContext->GetLibrary()->GetCommaSpaceDisplayString();
```

Comment:
Locale separator is produced on this line (above):
```
bool hasLocaleSeparator = Arrays::GetLocaleSeparator(szSeparator, &sepSize, Arrays::SeparatorBufferSize);
```
Also, this line is key to understanding how we use its size:
```
separator = JavascriptString::NewCopyBuffer(szSeparator, static_cast<charcount_t>(sepSize), scriptContext);
```